auction9-backend/handler.js

```diff
@@ -10,6 +10,13 @@ const mysql = require('serverless-mysql')({
   }
 });
 
+const header = () => {
+  return {
```

Comment:
I thought to make a function for example generateResponse that receive statusCode and body, header will be preset there. But this one also looks fine for me.

Reply:
@SvetislavMarjanovic  Sure, I will do that once I finish history.